rootfs/Dockerfile

```diff
@@ -37,6 +37,13 @@ RUN setcap cap_net_bind_service=+ep /usr/sbin/nginx \
 RUN mkdir -p /etc/ingress-controller/ssl /etc/ingress-controller/auth \
   && chown -R www-data.www-data /etc/nginx /etc/ingress-controller
 
+RUN mkdir -p /tmp/modsecurity/audit /tmp/modsecurity/upload \
```

Comment:
As this seems to be a recurring problem and probably not the last time permissions have to be adjusted, I'd be in favor of declaring an array of all directories used my the controller and applying ownership in one go. This mix and match of chown and file sed'ing feels suboptimal and prone to errors.

Reply:
Agree but we have a problem, some or most of the directories are located in the parent image or even worse in configuration files with some defaults. I don't see an easy way to tackle this.

Reply:
I persist in thinking defining an array in the Dockerfile instead of changing configs in place would be a good attempt at documenting what directories nginx/controller write to, at least.

Either that or move everything under /tmp, as long as it's consistent.

Reply:
done